logging/user-values-es-exporter.yaml

```diff
@@ -1,9 +1,9 @@
-# Chart: https://github.com/helm/charts/tree/master/stable/elasticsearch-exporter
-# Default values: https://github.com/helm/charts/blob/master/stable/elasticsearch-exporter/values.yaml
+# Helm Chart:     https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-elasticsearch-exporter
+# Default values: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-elasticsearch-exporter/values.yaml
 
 # image:
 #   repository: justwatch/elasticsearch_exporter
 #   tag: 1.1.0
 
 # es:
-#   uri: https://admin:admin@v4m-es-client-service.logging:9200
+#   uri: https://metricgetter:metricgetterpw@v4m-es-client-service.logging:9200
```

Comment:
Any way to use env vars and/or secrets for the URI?

Reply:
Based on the existing comment, I was showing 'sample' values...just pushed code to update comment to reflect the default behavior of setting this using env vars.